### ISSUE TITLE
Model mismatch occurs when training student network with own data set

### ISSUE THREAD

**Issue by rogressing** [TAMU-VITA/FasterSeg#88]
I used my own data set for training. According to the training steps given by the author, I first pre-trained the supernet and parameters, and then obtained the search file to train the teacher network. **After training the teacher network, I distilled it to the student network again, but when the student network is loaded, The problem occurs as follows**：
![image](https://github.com/VITA-Group/FasterSeg/assets/130099595/ea6d3024-7e69-4679-bb46-99f31b7ba074)
    And the prompt is that the loaded model and the current model do not match the prompt in the code here:
![image](https://github.com/VITA-Group/FasterSeg/assets/130099595/57971ffa-3a8f-4ce8-8385-4333741ca1ab)
    I would like to ask the author or other experts if they have encountered this kind of problem and how to solve it? And what I am very confused about is that the accuracy of my teacher's network training and verification is very low, only about 29%.

**Comment by rogressing** [comment]
I used my own data set for training. According to the training steps given by the author, I first pre-trained the supernet and parameters, and then obtained the search file to train the teacher network. After training the teacher network, I distilled it to the student network again, but when the student network is loaded, The problem occurs as follows：

**Comment by rogressing** [comment]
Thank you ,It's been solved
